src/rdkafka.h

```diff
@@ -7085,6 +7127,46 @@ typedef enum rd_kafka_AclOperation_t {
 /**@}*/
 
 
+/**
+ * @name ListOffsets API
+ * @brief Given a topic_partition list, provides the offset information.
+ * @{
+*/
+
+/**
+ * OffsetSpec for Earliest, Latest and Timestamp of the Offset to list via ListOffsets.
```

Comment:
* @enum rd_kafka_OffsetSpec_t
 * @brief Allows to specify the desired offsets when using ListOffsets.